app/controllers/mixins/vm_show_mixin.rb

```diff
@@ -167,6 +167,6 @@ def set_session_data
   end
 
   def breadcrumb_name(model)
-    ui_lookup_for_model(model || self.class.model.name).pluralize
+    ui_lookup(:models => model || self.class.model.name)
```

Comment:
@chessbyte : yes, that is confusing but it it not introduced by this PR. It's all over the code.